src/CoverageTransformer.js

```diff
@@ -91,7 +93,7 @@ class CoverageTransformer {
 			}
 		}
 
-		if (!match || !rawSourceMap) {
+		if (!rawSourceMap) {
```

Comment:
I'm not 100% sure if this is right this way... But i think this check is sufficient since we only need to have a source map (we dont care about a match... just throw an error if we could not find the source map)